Gruntfile.js

```diff
@@ -50,7 +50,25 @@ module.exports = function (grunt) {
     });
 
     walker.on('end', function () {
-      done();
+      var jsWalker = extract({
+        'input-dir': __dirname,
+        /* node_modules and test should not contain any strings
+         * Gruntfile causes an error and should contain no strings
+         * bin/server.js extracts "/", so it is excluded.
+         */
```

Comment:
*grumble* if the `.js` files were all in a `./lib` directory we woud not have this problem...